appgate/syncer/git.py

```diff
@@ -50,8 +50,6 @@
 github.com ssh-rsa AAAAB3NzaC1yc2EAAAABIwAAAQEAq2A7hRGmdnm9tUDbO9IDSwBK6TbQa+PXYPCPy6rbTrTtw7PHkccKrpp0yVhp5HdEIcKr6pLlVDBfOLX9QUsyCOV0wzfjIJNlGEYsdlLJizHhbn2mUjvSAHQqZETYP81eFzLQNnPHt4EVVUh7VfDESU84KezmD5QlWpXLmvU31/yMf+Se8xhHTvKSCZIFImWwoG6mbUoWf9nzpIoaSjB+weqqUUmpaaasXVal72J+UX2B+2RPW3RcT0eOzQgqlJL3RKrTJvdsjE3JEAvGq3lGHSZXy28G3skua2SmVi/w4yCE6gbODqnTWlg7+wC604ydGXA8VJiS5ap43JXiUFFAaQ==
 """
 
-GitVendor: TypeAlias = Literal["gitlab", "github"]
```

Comment:
Why remove this?

Reply:
Oops. I got lazy when I was fixing lint issues. Somewhere else in code, we get the type from `os.getenv()` and I need to figure out how to let mypy understand that helm chart would only supply those literals.

Reply:
Ah, here
https://github.com/appgate/sdp-operator/blob/c4a30b4317512429a3979fd6484697cbc637a640/appgate/syncer/operator.py#L86